Source/stores.cpp

```diff
@@ -969,6 +965,9 @@ void S_StartConfirm()
 	case STORE_SREPAIR:
 		strcpy(tempstr, "Are you sure you want to repair this item?");
 		break;
+	default:
+		strcpy(tempstr, "Are you sure?");

```

Comment:
Is this possible? I wouldn't create logic for impossible enum values. Either we should just fail somehow, or do nothing at best.

Adding a different string here may mislead people into thinking there is an extra behavior.

Reply:
Hm it was just to make it easy for anyone why would try to extend the stores later. I can remove it again if you think that's better.

Reply:
>  I can remove it again if you think that's better.

I think it is best we go for YAGNI here and avoid implementing "stub" things when we can.